### DISCUSSION TITLE
How to use af=alatency in mpv player and check if it's applied?

### DISCUSSION THREAD

**Discussion by NeilTohno** [mpv-player/mpv#14074] [Q&A] [answered]
[latency, alatency](https://ffmpeg.org/ffmpeg-all.html#toc-latency_002c-alatency)

> Measure filtering latency.
> 
> Report previous filter filtering latency, delay in number of audio samples for audio filters or number of video frames for video filters.
> 
> On end of input stream, filter will report min and max measured latency for previous running filter in filtergraph.

I found no conclusion (af=alatency) in the logs. 
Thanks

**Comment by richardpl** [comment, marked as answer]
Filter is not changing anything in its output its just reporting some numbers that for ordinary average users are irrelevant. No it does not measure A-V latency difference at all.
